services/ai_client.py

```
@@ -0,0 +1,51 @@
+        self.base_url = os.getenv("LEXERA_API_BASE_URL", "https://api.groq.com/openai/v1")
+        self.api_key = os.getenv("LEXERA_API_KEY", "")
+        self.model = os.getenv("LEXERA_MODEL", "llama-3.3-70b-versatile")
+        self.timeout = int(os.getenv("LEXERA_API_TIMEOUT", "60"))
+
+    @property
+    def configured(self) -> bool:
+        return bool(self.api_key)
+
+    def chat(self, system_prompt: str, user_prompt: str) -> str:
+        if not self.configured:
+            raise RuntimeError("LEXERA_API_KEY is not configured")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)